apps/test-suite/TestUtils.js

```diff
@@ -67,7 +68,8 @@ export function getTestModules() {
     modules.push(
       require('./tests/Contacts'),
       // require('./tests/SVG'),
-      require('./tests/Localization')
+      require('./tests/Localization'),
+      optionalRequire('./tests/NewNotifications')
```

Comment:
does this need to be `optionalRequire(() => require('./tests/NewNotifications'))`?